src/bot.py

```diff
@@ -198,18 +198,18 @@ def get_turn_points(self, train_idx, target_point, point_from=None):
         :param train_idx: int - train index
         :param target_point: int - target point index
         :param point_from: int - point index to build way from, if None - the way builds from current point
-        :return: list - list of turn points
+        :return: 2-tuple of list and int. List - list of turn points. Int - distance to target point
         """
         current_point = self.get_current_point(train_idx) if not point_from else point_from
-        point_to = self.dijkstra_algorithm(current_point)
+        point_to, route_weight = self.dijkstra_algorithm(current_point)
```

Comment:
point_to, trip_to = self.dijkstra_algorithm(current_point)

We use route term here for a sequence of turn points